tests/test_enterprise/test_signals.py

```diff
@@ -246,11 +248,213 @@ def test_default_content_filter(self, default_content_filter, expected_content_f
             assert enterprise_catalog.content_filter == expected_content_filter
 
 
+@mark.django_db
+class TestPendingEnterpriseAdminUserSignals(unittest.TestCase):
+    """
+    Test signals associated with PendingEnterpriseCustomerAdminUser.
+    """
+
+    def setUp(self):
+        """
+        Setup for `TestPendingEnterpriseAdminUserSignals` test.
+        """
+        self.admin_user = UserFactory(id=2, email='user@example.com')
+        self.enterprise_customer = EnterpriseCustomerFactory()
+        super(TestPendingEnterpriseAdminUserSignals, self).setUp()
+
+    def test_create_pending_enterprise_admin_user(self):
+        # verify that PendingEnterpriseCustomerUser record does not yet exist.
+        pending_ecus = PendingEnterpriseCustomerUser.objects.filter(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        )
+        self.assertFalse(pending_ecus.exists())
+
+        # create new PendingEnterpriseCustomerAdminUser
+        PendingEnterpriseCustomerAdminUserFactory(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        )
+
+        # verify that PendingEnterpriseCustomerUser record was created.
+        pending_ecus = PendingEnterpriseCustomerUser.objects.filter(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        )
+        self.assertTrue(pending_ecus.exists())
+
+    def test_delete_pending_enterprise_admin_user(self):
+        # create new PendingEnterpriseCustomerAdminUser
+        PendingEnterpriseCustomerAdminUserFactory(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        )
+
+        # verify that PendingEnterpriseCustomerUser record exists.
+        pending_ecus = PendingEnterpriseCustomerUser.objects.filter(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        )
+        self.assertTrue(pending_ecus.exists())
+
+        # delete the PendingEnterpriseCustomerAdminUser record and verify that the
+        # associated PendingEnterpriseCustomerUser is also deleted.
+        PendingEnterpriseCustomerAdminUser.objects.filter(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        ).delete()
+        pending_ecus = PendingEnterpriseCustomerUser.objects.filter(
+            user_email=self.admin_user.email,
+            enterprise_customer=self.enterprise_customer,
+        )
+        self.assertFalse(pending_ecus.exists())
```

Comment:
nit: could potentially refactor this assert does exist/does not exist check into a helper to reduce some duplication